bom/application/pom.xml

```diff
@@ -5843,12 +5823,12 @@
             </dependency>
             <dependency>
                 <groupId>org.infinispan</groupId>
-                <artifactId>infinispan-server-hotrod</artifactId>
+                <artifactId>infinispan-server-core</artifactId>
```

Comment:
@gsmet this should not be necessary (infinispan-server-core) unless it's for the <type>test-jar</type>